src/custom-element.js

```diff
@@ -1146,10 +1146,11 @@ function createBaseCustomElementClass(win) {
      *
      * Can only be called on a upgraded and built element.
      *
+     * @param {!AbortSignal} abortSignal
```

Comment:
Looks like other APIs such as `fetch` call it `signal` in their args. A nit, but it might read a bit better `signal.aborted`.